test/integration/retries.spec.js

```diff
@@ -2,56 +2,60 @@
 
 var assert = require('assert');
 var helpers = require('./helpers');
-var args = ['--reporter', 'dot'];
+var args = [];
 var bang = require('../../lib/reporters/base').symbols.bang;
 
 describe('retries', function() {
   it('are ran in correct order', function(done) {
-    helpers.runMocha('retries/hooks.fixture.js', args, function(err, res) {
-      var lines, expected;
+    helpers.runMocha(
```

Comment:
thanks prettier

Reply:
LOL. So it doesn't _just_ happen to me after all...

Reply:
"works as intended"